examples/java/pom.xml

```diff
@@ -372,7 +372,7 @@
           <groupId>org.codehaus.mojo</groupId>
           <artifactId>findbugs-maven-plugin</artifactId>
           <configuration>
-            <skip>true</skip>
+            <!-- skip>true</skip-->
```

Comment:
Go ahead and delete the whole `<plugin>` block, since it doesn't do anything now. Great!